src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs

```diff
@@ -289,6 +289,7 @@ public bool ConnectorAnchorViewModelExists
             {
                 connectorAnchorViewModelExists = value;
                 RaisePropertyChanged(nameof(ConnectorAnchorViewModelExists));
+                RaisePropertyChanged(nameof(PreviewState));
```

Comment:
why do we raise a different property change here?

Reply:
![image](https://user-images.githubusercontent.com/24754290/135417789-5daed851-db9e-4aab-a020-76678af2ef4a.png)
@QilongTang  Actually the `RaisePropertyChanged` call is what indicates that the wire should transition to `hover` mode. Open to suggestions on how to do it differently.

Reply:
Ah I missed this then, make sense